native/packages/react-native-bpk-component-theming/stories.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Component } from 'react';
+import {
+  StyleSheet,
+  View,
+  Platform,
+  Picker,
+} from 'react-native';
+
+import { storiesOf } from '@storybook/react-native';
+import { withTheme } from 'theming';
```

Comment:
Similarly this should come from our code, not from the `theming` dependency. There should be only one reference to `theming` in our code.